src/format.ts

```diff
@@ -1,6 +1,11 @@
-import { isString } from './utils.js'
+import { Params } from '.'
+import { isString } from './utils'
 
-function replaceHighlightTags(value, highlightPreTag, highlightPostTag) {
+function replaceHighlightTags(
+  value: string,
+  highlightPreTag?: string,
+  highlightPostTag?: string
+) {
```

Comment:
The return type is missing or is it infered based on the returned function?

Reply:
Yes, it's inferred. The return statement explicitly states `.toString()`.

However, there are other things which might not be perfect yet: Is the input param really always string as specified? Currently it looks like, but why would it need to check isString() in line 15 then?